src/schema/v2/artwork/index.ts

```diff
@@ -107,6 +122,9 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           return artistLoader(artist.id).catch(() => null)
         },
       },
+      artistInsights: {
```

Comment:
Not sure whether this is a good name for the field because these insights are insights for the given artwork..

Reply:
> these insights are insights for the given artwork 
Q: @olerichter00 are you sure this is correct? We name AuctionResults as ArtistInsightsAuctionResults (fragment ArtistInsightsAuctionResults_artist). Or is it something different?

Reply:
Well, it is actually a the artist insights for a specific medium, but I see your point, I will change it to `insights`